libs/galley-types/galley-types.cabal

```diff
@@ -66,7 +66,9 @@ library
   ghc-options:
     -O2 -Wall -Wincomplete-uni-patterns -Wincomplete-record-updates
     -Wpartial-fields -fwarn-tabs -optP-Wno-nonportable-include-path
-    -Wredundant-constraints -Wunused-packages
+    -Wredundant-constraints
+
+  -- -Wunused-packages
```

Comment:
Nit: is this supposed to be still left as a comment?

Reply:
Fixed, thanks for catching this.